docs/make.jl

```diff
@@ -12,6 +12,17 @@ GENERATED_BL_EXAMPLES = [
 ]
 GENERATED_BL_EXAMPLES = filter!(x -> isfile(x), GENERATED_BL_EXAMPLES)
 
+# Literate examples created in their own literate_example directory for now
+const examples_directory = pwd() * "/literate_examples/"
+const output_directory = pwd() * "/docs/src/generated/"
```

Comment:
- use `joinpath` instead of hardcoding slashes
- `pwd()` assumes you're running it from a specific directory: use `@__DIR__` instead

```suggestion
const examples_directory = joinpath(@__DIR__, "..", "literate_examples")
const output_directory = joinpath(@__DIR__, "src", "generated")
```